### PR TITLE
Run Docs tests on CI/CD

### LINKED ISSUE #6517: Setup IBM or prow jobs for the doc tests to run on every PR.

### PR DESCRIPTION
**Pull request by feloy**
**What type of PR is this:**
/kind tests

**What does this PR do / why we need it:**

Docs tests are executed on the same Kubernetes cluster as the Kubernetes integration tests, at the same time.

These docs tests are exectued one at a time, it should not impact the integration tests.

**Which issue(s) this PR fixes:**

Fixes #6517

**PR acceptance criteria:**

- [ ] Unit test 

- [ ] Integration test 

- [ ] Documentation 

**How to test changes / Special notes to the reviewer:**

### CODE REVIEW

File: .ibm/pipelines/kubernetes-tests.sh [hunk 1]
```diff
@@ -16,6 +16,7 @@ export SKIP_USER_LOGIN_TESTS=true
     echo Using Devfile proxy: ${DEVFILE_PROXY}
     make install
     make test-integration-cluster
+    make test-doc-automation
```
Comment (anandrkskd) [inline .ibm/pipelines/kubernetes-tests.sh:19]: @feloy why do we need to run `make test-doc-automation` on both file?

Comment (feloy) [reply .ibm/pipelines/kubernetes-tests.sh:19]: No, I'm checking on which one it passes. I'll remove one of them when the other one passes

Comment (anandrkskd) [reply .ibm/pipelines/kubernetes-tests.sh:19]: Understood, Thanks Philippe

**Review by rm3l — APPROVED** [review]

**Comment by feloy** [comment]
/override ci/prow/v4.11-integration-e2e
Not related

**Comment by feloy** [comment]
/override ci/prow/v4.11-integration-e2e
Flaky e2e test

**Pull request merged**